diff --git a/apps/system/js/window_manager.js b/apps/system/js/window_manager.js
index 1252ad7..f064ee1 100644
--- a/apps/system/js/window_manager.js
+++ b/apps/system/js/window_manager.js
@@ -378,8 +378,10 @@ var WindowManager = (function() {
       'Messages',
       // Crashes when launched OOP (bug 775997)
 
-      'Settings'
+      'Settings',
       // Bluetooth is not remoted yet (bug 755943)
+      
+      'NativePush'
     ];
 
     if (outOfProcessBlackList.indexOf(name) === -1) {
